string label
objectDetection[] detections
float32 mid_height